vtkext/private/module/vtkF3DRenderer.cxx

```diff
@@ -1838,6 +1861,14 @@ void vtkF3DRenderer::Render()
 {
   this->ConfigureJitter(this->AntiAliasingModeEnabled == vtkF3DRenderer::AntiAliasingMode::TAA);
 
+  if (int* size = this->GetSize();
+      CurrentViewportSize[0] != size[0] || CurrentViewportSize[1] != size[1])
```

Comment:
@Meakk is this the right way to go with this or is there a resize callback we could use ?